hoomd/hpmc/update.py

```diff
@@ -736,7 +736,8 @@ class QuickCompress(Updater):
         trigger (hoomd.trigger.trigger_like): Update the box dimensions on
             triggered time steps.
 
-        target_box (hoomd.box.box_like): Dimensions of the target box.
+        target_box (hoomd.box.box_like or hoomd.variant.box.BoxVariant):
```

Comment:
```suggestion
        target_box (`hoomd.box.box_like` or `hoomd.variant.box.BoxVariant`):
```
I think this is necessary for the links to work correctly.